dbt_transformation_create.go

```diff
@@ -0,0 +1,119 @@
+package fivetran
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+type DbtTransformationCreateService struct {
+	c          *Client
+	dbtModelId *string
+	schedule   *DbtTransformationSchedule
+	runTests   bool
```

Comment:
It should be *bool because it's nullable value. We can't omit non-nullable boolean, otherwise it should have default value.
Let's follow the same approach we have for connector/destination with *bool type

Reply:
Done. :)